firejail/virtualbox.profile

```diff
@@ -0,0 +1,22 @@
+# virtualbox profile
```

Comment:
I think most firejail files are also installed by dpkg, hashbang only customized some of them. Do we really need to carry the entire dir?